nodes/fabric.js

```diff
@@ -74,12 +74,35 @@ module.exports = function (RED) {
      * @param {Node} node node
      * @returns {Promise<Buffer>} promise
      */
-    function evaluate (contract, payload, node) {
-        node.log(`evaluate ${payload.transactionName} ${payload.args}`);
-        return contract.submitTransaction(payload.transactionName, ...payload.args);
+    function evaluate(contract, payload, node) {
+        node.log(`evaluate ${payload.transactionName} ${payload.transactionArgs}`);
+        return contract.submitTransaction(payload.transactionName, ...payload.transactionArgs);
+    }
+
+    /**
+     * @param {object} channel the channel object
+     * @param {object} payload payload
+     * @param {string} contractName the name of the contrat
+     * @param {Node} node node
+     * @returns {Promise<Buffer>} promise
+     */
+    function query(channel, payload, contractName, node) {
```

Comment:
you don't need this it does the same as evaluate

Reply:
True. I did not pay attention to that. I'll remove the query.